Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM coturn/coturn
+USER root
```

Comment:
Why are we setting this?

Reply:
As far as the issues I have encountered, the Coturn .pid file is written to `/var` by default, which allows it to do so; otherwise, it falls back to `/tmp`. Haven't observed other issues when not setting the user yet.

Reply:
The base image does not set this, how are they handling that differently?